<?xml version="1.0" encoding="utf8"?>

<launch>
  <node pkg="robot_control" type="avoid_obstacle_03.py" name="avoid_obstacle_03" output="screen">
    <param name="param_path" value="$(find robot_control)/params/params.yaml"/>
  </node>
</launch>
